frontend/src/api/url.js

```diff
@@ -0,0 +1,8 @@
+const API_URL = `${import.meta.env.VITE_API_URL}/api/`;
```

Comment:
Myślę, że '/api/' nie powinno być zhardcodowane. Cały adres backendu powinien być konfigurowalny tak, aby móc to podmieniać bez potrzeby re-deploymentu. Przy obecnym deploymencie i tak musimy przebudować obraz, ale docelowo gdybyśmy nie mieli tego w kontenerze to można by to było podmienić prawie, że realtime.